### PR TITLE
[docs] describes -dev option

### PR DESCRIPTION
**Pull request by MargaritaLoseva**
refer to #2545

### CODE REVIEW

File: docs/articles/documentation/using-testcafe/programming-interface/createtestcafe.md [hunk 1]
```diff
@@ -9,20 +9,21 @@ checked: true
 Creates a [TestCafe](testcafe.md) server instance.
 
 ```text
-async createTestCafe([hostname], [port1], [port2]) → Promise<TestCafe>
+async createTestCafe([hostname], [port1], [port2], developmentMode) → Promise<TestCafe>
```
Comment (miherlosev) [inline docs/articles/documentation/using-testcafe/programming-interface/createtestcafe.md]: The correct spelling  - `async createTestCafe([hostname], [port1], [port2], sslOptions, developmentMode) → Promise<TestCafe>`.

File: docs/articles/documentation/using-testcafe/programming-interface/createtestcafe.md [hunk 2]
```diff
@@ -9,20 +9,21 @@ checked: true
...
 `port1`, `port2`&#160;*(optional)* | Number | Ports that will be used to serve tested webpages.                                                                                                                                                            | Free ports selected automatically.
+`developmentMode`&#160;*(optional)* | Boolean | Enables/disables mechanisms to log and diagnose errors. | `false`
 
 **Example**
 
 ```js
 const createTestCafe = require('testcafe');
 
-createTestCafe('localhost', 1337, 1338)
+createTestCafe('localhost', 1337, 1338, true)
```
Comment (miherlosev) [inline docs/articles/documentation/using-testcafe/programming-interface/createtestcafe.md]: `createTestCafe('localhost', 1337, 1338, sslOptions, true)`

**Comment by AndreyBelym** [comment]
I think we should merge the PR about HTTPS before processing this PR.

**Comment by MargaritaLoseva** [comment]
Yes, let's merge this PR after the PR about HTTPS.

File: docs/articles/documentation/using-testcafe/programming-interface/createtestcafe.md (hunk 2 above)
Comment (MargaritaLoseva) [reply docs/articles/documentation/using-testcafe/programming-interface/createtestcafe.md]: Yes, we'll merge this string with a string from the PR about HTTPS

**Pull request merged**